CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to Round Manager and Grant Explorer
+
+Thank you for your desire to contribute to Gitcoin! Below you will find some ways to help, but please don't feel limited by them, and if you have any other ideas, pop by our [Discord](https://discord.gg/gitcoin)
+
+## Reporting issues
+
+Before reporting an issue, please make sure to verify the following:
+- You're running the latest version, including other components and systems the project might interact with
+- Your dependencies are up to date
+
+If the issue still persists, search the issue tracker. If there is an open ticket, please don't comment '+1' or similar - use the '👍' emoji. Only comment if you have additional context, reproduction instructions or anything that might help resolve the issue.
+
+When opening a new issue, make sure it contains the following:
+- Step-by-step reproduction instructions
+- Screenshots or videos of the issue, if applicable
+- Version information of the project
+- Any other context that you feel would help resolve the issue
+
+## Contributing code
+
+We welcome all code contributions, large or small. Please see [DEV.md](packages/round-manager/docs/DEV.md) for instructions on how to set up the project for development.
+
+Before submitting a PR, please make sure that all tests, linting and formatting rules are passing.
+
+We squash all PRs by default. You can request your PR commits to not be squashed if the're well named and organized, and it would make sense to preserve their context.
+
+_To keep the repository clean, we may mark PRs as stale if there has been no activity for more than a week, which are then subject to closure and deletion. If you are stuck, feel free to reach out for help in [Discord](https://discord.gg/gitcoin)._ 
```

Comment:
It might be worth mentioning that PRs which are incomplete for longer than 2 weeks will be closed !

Reply:
chatted with Josef -- let's keep the guide as-is for now, and we can revisit adding new terms if we run into issues with external contributors keeping PRs open for excessively long times